src/applications/claims-status/tests/e2e/08.claim-letters-keyboard-only.cypress.spec.js

```diff
@@ -3,18 +3,21 @@ import claimLetters from './fixtures/mocks/claim-letters/list.json';
 
 describe('Claim Letters Page', () => {
   beforeEach(() => {
-    cy.intercept('GET', '/v0/claim_letters', claimLetters.data);
+    cy.intercept('GET', '/v0/claim_letters', claimLetters.data).as(
+      'lettersLoaded',
```

Comment:
Does this alias make a functional difference? Wondering if it was intentionally added